eng/pipelines/templates/build-pull-request.yml

```diff
@@ -39,7 +39,7 @@ jobs:
     displayName: Authenticate NuGet
 
   # Runs the full build of the projects in the repository. See Build.proj for details.
-  - script: $(Build.SourcesDirectory)/build.cmd /v:normal /p:Configuration=$(BuildConfiguration) /p:CIBuild=true
+  - script:  $(Build.SourcesDirectory)/build.cmd /v:normal /p:Configuration=$(BuildConfiguration) /p:CIBuild=true /p:Test=true
```

Comment:
you could remove "/p:Test=true" because it is the default settings

Reply:
same for the build-official-release

Reply:
done